src/extensions/cli-extension.ts

```diff
@@ -21,7 +23,59 @@ const disneyApi: ApisauceInstance = http.create({
   },
 });
 
-// Retrieve a basic auth token that is required for requests to their dining availability endpoint
+/**
+ * Returns Restaurant Information (Real Names to IDs and such)
+ * @param {ApisauceInstance} api
+ * @returns {Promise<any>}
+ */
+async function restaurantMapping(api: ApisauceInstance = disneyApi): Promise<any> {
+  api.setHeader('x-requested-with', 'XMLHttpRequest');
+  const response = await api.get('/dining/');
+  const data = <string>response.data;
+  const dom = new JSDOM(data);
+
+  const finderBlob = dom.window.document.querySelector('#finderBlob').innerHTML;
+  const PEP = <any>{};
+  //TODO: This is sort of dangerous but its the only way to get what we want from the blob
+  eval(finderBlob);
+
+  //Scan the DOM for the real name
+  return Object.values(PEP.Finder.List.cards).reduce((prev: any, card: any) => {
```

Comment:
Suggestion. Instead of returning this object, I suggest filtering out only the reservation based restaurants. 
I tried this to no avial: await api.get('/dining/#/reservations-accepted/');

here is my method :
```

  //Scan the DOM for the real name
  var locationsobj =  Object.values(PEP.Finder.List.cards).reduce((prev: any, card: any) => {
    const htmlCard = dom.window.document.querySelector(card.ref);
    return {
      ...prev,
      [card.id.split(';')[0]]: { ...card, displayName: htmlCard.querySelector('.cardName').innerHTML },
    };
  }, {});
  //Keep only the reservation locations
  const locationsArr = Object.entries(locationsobj);
  const locationsAdjusted = locationsArr.filter(([key, value]) => value['facets']['reservations-accepted'] !== undefined)
                                      .filter(([key, value]) => value['facets']['reservations-accepted']['value']%2 !== 0);
  return Object.fromEntries(locationsAdjusted);
```

Reply:
Looks good. I'll add this in the next week and additionally do a regular expression to pull out PEP instead of eval-ing it